backend-lambda/package.json

```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "pg": "^8.11.5",
     "pg-format": "^1.0.4",
-    "prettier": "^3.3.2"
+    "prettier": "^3.3.2",
+    "sam": "^0.0.7"
```

Comment:
Can sam be a devDependency? If so, it should be.

Reply:
Not a standard practice for SAM apparently, removing from the package.json